### DISCUSSION TITLE
`BURN_RATE` in `CHID_hrr.csv`

### DISCUSSION THREAD

**Discussion by TristanHehnen** [firemodels/fds#13335] [General]
Hi,

In section "27.2 Heat Release Rate and Related Quantities" in the User's Guide (FDS-6.9.1-0-g889da6a-release), the `BURN_RATE` is discussed as part of the `CHID_hrr.csv` file 
I could not fine any other instance of `BURN_RATE` anywhere in the User's Guide than in this section.
https://github.com/firemodels/fds/blob/23a3fd6d69f90cbd5c3907a877eac9db79b0a7ac/Manuals/FDS_User_Guide/FDS_User_Guide.tex#L14276

If I run simulations with a gas burner (`HRRPUA`) or pyrolysis, the "correct" mass loss rate is provided, for example "MLR_PROPANE" for a gas burner. 

Is the `BURN_RATE` in the `CHID_hrr.csv` file a correct column header and description?

Tristan

**Comment by mcgratta** [comment]
This is probably a term left over from the past when we had only one fuel and the `BURNING RATE` and `MASS LOSS RATE` were the same thing. I'll fix.

**Comment by mcgratta** [comment]
[I cleaned out these sections containing old information.](https://github.com/firemodels/fds/pull/13338) 

There is a better description of these quantities and now this section points to it.

**Reply by TristanHehnen** [reply]
Thank you very much!
